packages/Php/tests/Rector/FuncCall/RemoveExtraParametersRector/Fixture/func_get_arg.php.inc

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Rector\Php\Tests\Rector\FuncCall\RemoveExtraParametersRector\Fixture;
+
+class FuncGetArg
+{
+    public function __construct() {
+        self::("aa", "bb", "cc");
+    }
+    
+    static public function dumpArgs() {
+        $arg_list = func_get_args();
+        var_dump($arg_list);
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Php\Tests\Rector\FuncCall\RemoveExtraParametersRector\Fixture;
+
+class FuncGetArg
+{
+    public function __construct() {
+        self::("aa", "bb", "cc");
```

Comment:
I expect rector not to strip this call-site arguments, because the method beeing called uses some `func_get_args` magic to handle them